data/projectRouter.js

```diff
@@ -0,0 +1,92 @@
+const express = require( 'express' );
+const Projects = require( './helpers/projectModel' );
+const router = express.Router();
+
+//RECIEVING ALL PROJECTS
+router.get( '/' , async ( req, res ) => {
+    try {
+        const projects = await Projects.get( req.query.id );
+        if ( projects ) {
+            res.status( 200 ).json( projects );
+        } else {
+            res.status( 404 ).json({ message: 'Project not found' })
+        }
+    } catch ( error ) {
+        console.log( error );
+        res.status( 500 ).json({ message: 'Error getting the projects' });
+    }
+});
+
+//RECIEVING THE PROJECTS BY ID
+router.get( '/:id' , async ( req, res ) => {
+    try {
+        const projects = await Projects.get( req.query.id );
+        if ( projects ) {
+            res.status( 200 ).json( projects );
+        } else {
+            res.status( 404 ).json({ message: 'Project not found' })
+        }
+    } catch ( error ) {
+        console.log( error );
+        res.status( 500 ).json({ message: 'Error getting the projects' });
+    }
+});
+
+//ADD A PROJECT
+router.post( '/', async ( req, res ) => {
+    console.log( req.body );
```

Comment:
you may want to sanitize the data coming in from the user here, you could pull off the keys needed and ensure that only what you are wanting is going in,   const { name, description, completed } = req.body and in the insert await Projects.insert({name, description, completed})  this will make sure you only pass those 3 properties to your db no matter how much stuff the client sends you in the req.body